cmd/connectivity_pod_to_pod.go

```diff
@@ -30,19 +29,19 @@ func newConnectivityPodToPodCmd() *cobra.Command {
 				return errors.Errorf("provide both SOURCE_POD and DESTINATION_POD")
 			}
 
-			fromPod, err := kuberneteshelper.PodFromString(args[0])
+			sourcePod, err := pod.FromString(args[0])
```

Comment:
For consistency w/ the rest of the repo, can we change `sourcePod` to `srcPod` ?